### ISSUE TITLE
Context endpoint on API does not output included and excluded technologies

### ISSUE THREAD

**Issue by sethers** [zaproxy/zaproxy#6001]
**Describe the bug**
When browsing the context through the API, it does not output what technologies are included or excluded.  The API correctly allows a user to include or exclude technologies programmatically, however the view Context  endpoint is missing data from its output. 

**To Reproduce**
1. Open ZAP api in your browser, http://localhost:8080/UI/context/view/context/
2. Enter in a valid context name and the api key.
3. The results will not show you the included or excluded technologies

**Expected behavior**
A list of the included or excluded technologies.

** To fix **
This method is missing the output of the included and excluded technologies
https://github.com/zaproxy/zaproxy/blob/master/zap/src/main/java/org/zaproxy/zap/extension/api/ContextAPI.java#L385

**Comment by thc202** [comment]
There are endpoints to get that information though, `excludedTechnologyList` and `includedTechnologyList`.

**Comment by shankeerthan-kasilingam** [comment]
Hi @kingthorin and @thc202,
I'm new to zaproxy. Can i try to fix this issue?

**Comment by kingthorin** [comment]
Absolutely! 

Dev docs are here:https://www.zaproxy.org/docs/developer/
Relevant code: https://github.com/zaproxy/zaproxy/blob/main/zap/src/main/java/org/zaproxy/zap/extension/api/ContextAPI.java

**Comment by shankeerthan-kasilingam** [comment]
I fixed the issue and tested it. Sorry I missed to sign-off of the commit. Would it be ok to create a new PR with signoff? (The current PR is blocked by DCO https://github.com/zaproxy/zaproxy/pull/7194)

**Comment by kingthorin** [comment]
There’s no need for a new PR. Just do an interactive rebase and squash it all into a single commit, sign it off, force push it.
